### PR TITLE
Remove python 2 requirements install from docker images.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/python/system/archive.py [hunk 1]
```diff
@@ -295,14 +290,8 @@ def unpack(archive_path,
       logs.log_error(
           'Failed to extract everything from archive %s.' % archive_filename)
 
-  elif archive_type == ArchiveType.TAR or archive_type == ArchiveType.TAR_LZMA:
+  elif archive_type in [ArchiveType.TAR, ArchiveType.TAR_LZMA]:
```
Comment (oliverchang) [inline src/python/system/archive.py:293]: nit: use a tuple instead of list. i.e. `c in (A, B)`.

Comment (inferno-chromium) [reply src/python/system/archive.py]: Done!

**Pull request merged**